### PR TITLE
4.3.2 PR

### PR DESCRIPTION
**Pull request by bilalsattar24**

### CODE REVIEW

File: js/yahoo/matchupsPage.js [hunk 1]
```diff
@@ -52,16 +52,16 @@ const week_to_date = {
...
-  "20": "2020-3-2",
-  "21": "2020-3-9",
-  "22": "2020-3-16",
-  "23": "2020-3-23",
-  "24": "2020-3-30",
-  "25": "2020-4-6",
-  "26": "2020-4-13",
-  "27": "2020-4-20",
-  "28": "2020-4-27"
+  "18": "2020-2-24",
```
Comment (bilalsattar24) [inline js/yahoo/matchupsPage.js]: everywhere across our code we are referring to "week 17 and 18" as week 17 and then skipping to week 19 afterwards. does it make sense to remove week 18 from this? or are you handling it elsewhere?

Comment (alexanderDoria) [reply js/yahoo/matchupsPage.js]: we get date on the Matchups page from the url of the page OR the current date if not in the url. After the ASB, the week in the url and the week on the page won't be in sync. I changed it to reflect the week in the url.

File: js/yahoo/matchupsPage.js [hunk 2]
```diff
@@ -1132,11 +1131,14 @@ function calculateStats(data, cat) {
 function isDateSelected() {
   var bar = document.getElementById("yspmaincontent").children[0].children[0]
     .children[0].children[1].children[0].children[0];
+  //console.log("bar: ", bar);
   for (var i = 1; i < bar.children.length; i++) {
+    //console.log("bar[i]: ", bar.children[i]);
     if (
       bar.children[i].getAttribute("class").includes("Selected") &&
-      !bar.children[i].innerText.includes("Matchup Totals")
+      !bar.children[i].innerText.includes("Matchup")
```
Comment (bilalsattar24) [inline js/yahoo/matchupsPage.js]: whats this checking for?

Comment (alexanderDoria) [reply js/yahoo/matchupsPage.js]: I believe you wrote this. This is checking to make sure the dates aren't selected on the top bar of the Matchups page.

**Pull request merged**